src/framework/tours/view/toursprovider.cpp

```diff
@@ -78,7 +78,8 @@ void ToursProvider::doShow()
     m_currentStep++;
 
     const ui::INavigationControl* parentControl = findControl(step.controlUri);
-    IF_ASSERT_FAILED(parentControl) {
+    if (!parentControl) {
```

Comment:
Maybe we need to add a warning to the log here... or at least a comment in the code, why this might happen

Reply:
ok, added